src/hotspot/share/opto/vectorIntrinsics.cpp

```diff
@@ -425,6 +434,15 @@ bool LibraryCallKit::inline_vector_shuffle_iota() {
 // VM shuffleToVector(Class<VM> VecClass, Class<?>E , Class<?> ShuffleClass, Sh s, int length,
 //                    ShuffleToVectorOperation<VM,Sh,E> defaultImpl)
 bool LibraryCallKit::inline_vector_shuffle_to_vector() {
+  // TODO shuffle is not supported on SVE
+  // we SHOULD move these codes after implementing shuffle
+  if (Matcher::supports_scalable_vector()) {
+    if (C->print_intrinsics()) {
+      tty->print_cr("  ** shuffle to vector not supported on SVE");
```

Comment:
I think it's a bit intrusive to do a workaround here, which will be removed soon, for SVE implementation in shared code.